docs/docs/troubleshooting/errors/INVALID_CHAT_HISTORY.md

```diff
@@ -0,0 +1,30 @@
+# INVALID_CHAT_HISTORY
+
+This error is raised in the prebuilt [create_react_agent][langgraph.prebuilt.chat_agent_executor.create_react_agent] when the `call_model` graph node receives a malformed list of messages. Specifically, it is malformed when there are `AIMessages` with `tool_calls` that do not have a corresponding `ToolMessage`.
```

Comment:
Could be good to clarify:

`tool_calls` (representing requests to invoke tools) do not have a corresponding `ToolMessage` (representing results of tool call invocation)

Reply:
`ToolMessage` contains results of a single tool call (and has a corresponding tool call ID)